src/openms/source/FORMAT/MzTabFile.cpp

```diff
@@ -2866,10 +2908,11 @@ namespace OpenMS
       Size n_best_search_engine_score = mz_tab.getMetaData().protein_search_engine_score.size();
 
       // add header
-      out.push_back(generateMzTabProteinHeader_(protein_section[0], n_best_search_engine_score, mz_tab.getProteinOptionalColumnNames()));
+      // use the last element as a reference row, since they contain group abundance data if set
```

Comment:
why use the last element? maybe add another comment as it kind of seems like a hack we needed to introduce